app/src/main/java/co/condorlabs/customcomponents/customsignature/readme.md

```diff
@@ -0,0 +1,19 @@
+# Signature
+Emerald provides a signature dialog.
```

Comment:
this should required information about the listener I have to implement to get the signature